src/components/About.js

```diff
@@ -72,7 +80,7 @@ const About = (props) => {
   return (
     <Paneset>
       <Pane defaultWidth="30%" paneTitle="User interface">
-        <h4>Foundation</h4>
+        <Headline>Foundation</Headline>
```

Comment:
I see <Headline> defaults to an <h2> and these instances are replacing <h4>'s.  I haven't looked into the current structure of the about page, so it may not make a difference, but is switching to <h2> here intentional?